### PR TITLE
updating dependencies and angular beta.16

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "pack": "./crxmake.sh"
   },
   "dependencies": {
-    "angular2": "2.0.0-beta.15",
+    "angular2": "2.0.0-beta.16",
     "basscss": "^7.0.4",
     "core-js": "^2.2.2",
     "crypto": "0.0.3",
```
Comment (vanessayuenn) [inline package.json]: no `zone.js` update?

Comment (sumitarora) [inline package.json]: There is issue when updating created one: https://github.com/rangle/augury/issues/352

Comment (vanessayuenn) [inline package.json]: 👌

**Comment by vanessayuenn** [comment]
🐑 🇮🇹

**Pull request merged**